openmc/region.py

```diff
@@ -221,6 +222,12 @@ def apply_operator(output, operator):
         # at the end
         return output[0]
 
+    @abstractmethod
+    def clone(self):
+        """Create a copy of this region - each of the surfaces in the
+        region's nodes will be cloned and will have new unique IDs."""
+        return False
```

Comment:
I agree. I stubbed this out when implementing this and forgot to properly implement it.